kafka-ui-react-app/src/components/common/Checkbox/Checkbox.tsx

```diff
@@ -8,14 +8,15 @@ interface CheckboxProps {
   name: string;
   label: React.ReactNode;
   hint?: string;
+  cursor?: string;
 }
 
-const Checkbox: React.FC<CheckboxProps> = ({ name, label, hint }) => {
+const Checkbox: React.FC<CheckboxProps> = ({ name, label, hint, cursor }) => {
   const { register } = useFormContext();
 
   return (
     <div>
-      <InputLabel>
+      <InputLabel cursor={cursor}>
```

Comment:
Label should not have pointer cursor

Reply:
Roma told me several times that I need to make on label cursor  pointer